.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,53 @@
+<!--
+Thank you for creating this pull request!
+
+Provide a general summary of your changes in the Title above.
+And please provide enough information in the description below, so that others can review your pull request.
+-->
+
+## Pull Request Type
+
+- [ ] I have checked there is no other PR open for the same change.
+
+This is a:
+- [ ] Bug fix
+- [ ] New feature
+- [ ] QA update
```

Comment:
I'm not sure how `QA update` is supposed to be interpreted. The way I understand it, quality assurance is about verifying that changes that were implemented are properly distributed and actually do what they are supposed to do.

We might just have a different understanding of QA, but is this meant to signify "code quality improvement" maybe?
```suggestion
- [ ] Code quality improvement
```

Reply:
I agree, making this more descriptive will prevent confusion. Fixed.